serverless/webpack.config.js

```diff
@@ -1,7 +1,36 @@
 const serverlessWebpack = require('serverless-webpack');
+const babelOptions = require('../babel.config');
+const path = require('path');
 
 module.exports = {
   entry: serverlessWebpack.lib.entries,
   target: 'node',
   mode: 'production',
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx|ts|tsx)$/,
+        exclude: /node_modules/,
+        use: {
+          loader: require.resolve('babel-loader'),
+          options: {
+            ...babelOptions,
+          },
+        },
+      },
+    ],
+  },
+  optimization: {
+    // We do not want to minimize our code.
+    minimize: false,
+  },
+  resolve: {
+    extensions: ['*', '.js', '.jsx', '.ts', '.tsx'],
```

Comment:
What does `'*'` do?

Reply:
yeah fair point, ill remove it, pretty sure it does.. nothing.